pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Run command:
+
+`ansible {{group}} -m command '{{command}}'`
+
+- Run command with sudo:
+
+`ansible {{group}} -bKm command '{{command}}'`
+
+- Ping:
+
+`ansible {{group}} -m ping`
+
+- Get system info:
+
+`ansible {{group}} -m setup`
+
+- List computers belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Using a custom inventory file:
```

Comment:
I added this command because my workflow uses this specific option a lot. I have multiple Ansible projects, each having their own inventory file.

I'm not sure if this assumption applies to everyone else, but I felt that the amount of documentation here was still manageable.